docs/source/get_involved/code_of_conduct.rst

```diff
@@ -83,6 +109,7 @@ hope you will consider to be appropriate community guidelines:
 -  **Be respectful of different viewpoints and experiences**. Do not
    engage in homophobic, racist, transphobic, ageist, ableist, sexist,
    or otherwise exclusionary behaviour.
+   In disagreements, assume best intentions.
```

Comment:
what is this concretely? Sorry but it sounds a bit like random "bla bla"? :)

Reply:
Same as "be respectful of different viewpoints". This is a guiding principle, examples are spelled out in appendix A and further below.